packages/kit/src/exports/vite/dev/index.js

```diff
@@ -27,6 +29,15 @@ const cwd = process.cwd();
 export async function dev(vite, vite_config, svelte_config) {
 	installPolyfills();
 
+	const async_local_storage = new AsyncLocalStorage();
```

Comment:
this has been added in node 18.16 (if the node homepage is to believed) but kit currently has "node": ">=18.13" in its package.json. So technically a breaking change

Reply:
Where are you seeing that? This is from https://nodejs.org/api/async_context.html#class-asynclocalstorage

<img width="624" alt="image" src="https://github.com/sveltejs/kit/assets/1162160/a4514ba0-a7bd-4908-834c-9a449f4b1b1c">

Reply:
didn't expand the history and mistook the information for bind below for the class...